multicluster/controllers/multicluster/core/resourceimport_controller.go

```diff
@@ -160,25 +174,27 @@ func (r *ResourceImportReconciler) handleResImpUpdateForService(ctx context.Cont
 		return ctrl.Result{}, err
 	}
 	svcImpObj := getMCServiceImport(resImp)
+	if !reflect.DeepEqual(*svc, corev1.Service{}) {
```

Comment:
Should we just check svc.Spec.ClusterIP?

Reply:
done